profile.py

```diff
@@ -150,8 +172,10 @@ def right_penumbra(self, upper=0.9, lower=0.1):
     def center(self, level=0.5):
         return 0.5 * (self.x_at_y(level) + self.x_at_y(level, reverse=True))
 
-    def mirror(self):
-        self.y = self.y[::-1]
+    def mirror(self, m=0):
+        # self.y = 2*m - self.y
+        self.x = 2*m - self.x
+        self.fix_profile()
```

Comment:
Do you really need to sort profile using sorting method ?
Maybe this might be helpful (a is numpy array):
a = a[::-1]